synapse/python_dependencies.py

```diff
@@ -78,8 +78,6 @@
     "matrix-synapse-ldap3": ["matrix-synapse-ldap3>=0.1"],
     # we use execute_batch, which arrived in psycopg 2.7.
     "postgres": ["psycopg2>=2.7"],
-    # ConsentResource uses select_autoescape, which arrived in jinja 2.9
-    "resources.consent": ["Jinja2>=2.9"],
```

Comment:
Will this mean someone installing `matrix-synapse[resources.consent]` will now break? Not sure if we care, but a thing to consider!

Reply:
Huh, well I thought it would but apparently `pip` just defaults to installing the base dependencies instead :tada: